### DISCUSSION TITLE
Browser Containers for qutebrowser

### DISCUSSION THREAD

**Discussion by s-praveen-kumar** [qutebrowser/qutebrowser#7325] [Show and tell] [answered]
I created a simple set of scripts to get browser containers similar to the one in Firefox.
This allows multiple containers for multiple accounts or to prevent cookie tracking across sites.

Check it out here : [qute-containers](https://github.com/s-praveen-kumar/qute-containers)

**Comment by The-Compiler** [comment, marked as answer]
This got lost a bit after my holidays, whoops! I haven't tried it, but it looks pretty cool. I added it to the list of userscript repos (21c82b029b141fe9d89946b0c9378538ff395b26) and also mentioned it in the relevant issue (#4102).
